modules/mobile/spec/request/appointments_list_request_spec.rb

```diff
@@ -132,11 +132,11 @@
           it 'has the correct links with no prev' do
```

Comment:
These changes are all because the pagination helper orders the params differently.